Source/WebCore/dom/Document.h

```diff
@@ -2597,6 +2600,8 @@ class Document
     bool m_scheduledDeferredAXObjectCacheUpdate { false };
     bool m_wasRemovedLastRefCalled { false };
 
+    bool m_isInitialAboutBlank { false };
```

Comment:
The FrameLoader already has this state:
bool isShowingInitialAboutBlank = m_localFrame->loader().stateMachine().isDisplayingInitialEmptyDocument()

Reply:
At the location where that code is used the information is valid, however the state quickly moves on to FirstLayoutDone, so I feel we still need to store it somewhere so the information remains static (until next navigation). The latest upload tries to do that, PTAL.